lib/rules/object-curly-spacing.js

```diff
@@ -67,7 +67,7 @@ module.exports = function(context) {
     * @returns {void}
     */
     function reportRequiredBeginningSpace(node, token) {
-        context.report(node, token.loc.start,
+        context.report(node, token.loc.end,
```

Comment:
I think we should not make the change here. Because at that time it will actually show the wrong location to put space.